include/SphereSceneNode.h

```diff
@@ -23,45 +23,20 @@
 
 /******************************************************************************/
 
+const int sphereLods = 5;
```

Comment:
did you mean to declare a global in a header file? Can this be scoped within some class?

Reply:
Changed